src/index.html

```diff
@@ -11,7 +11,140 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <main>
+      <form autocomplete="off" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+        <fieldset>
+          <legend>Personal information</legend> 
+          <div class="div-class">
+            <label>
+              Surname:
+              <input type="text"  name="surname" 
+                minlength="2" maxlength="50"
+              >
```

Comment:
check it for each element
```suggestion
              <input 
                type="text" 
                name="surname" 
                minlength="2" 
                maxlength="50"
              >
```